### DISCUSSION TITLE
Collisional ionization

### DISCUSSION THREAD

**Discussion by SillyTom** [SmileiPIC/Smilei#770] [Q&A]
Does Smilei support calculating the collisional ionization rate between electrons and neutral atoms?

**Comment by mccoys** [comment]
Yes the ionization rate is calculated for all ionization states, including neutral atoms. There is one important approximation though: the rate assumes that all atoms are in the ground state

**Reply by SillyTom** [reply]
Dear developer, are you sure you are talking about the collisional ionization (not the field ionization)? 

I have this doubt because the doc says "The binary collisions can also be ionizing **if they are electron-ion collisions**. The approach is almost the same as that provided in [[Perez2012]]."     https://smileipic.github.io/Smilei/Understand/collisions.html#collisional-ionization

Besides, in my simulation, I also find little difference in the results when I change the neutral atom density.

So, does **collisional** ionization occurs between electrons and neutral atoms in Smilei?

Thanks for your help, this is quite important to me.

**Reply by mccoys** [reply]
Yes I am referring to collisional ionization. To be honest, I never tried with neutral atoms, but I see no reason why it wouldn't work. Whereas the collision model currently cannot treat neutrals (I am working on it), the collisional ionization model can. The only thing that might cause a problem would be an approximation in the binding energies leading to a wrong rate with neutrals but I don't see it. 

Did you check whether the ionization was incorrect vs theory?

**Reply by mccoys** [reply]
In fact, this figure starts with neutrals https://smileipic.github.io/Smilei/Understand/collisions.html#ionizationmultiple
